osu.Game.Tests/Visual/Editing/TestSceneEditorSaving.cs

```diff
@@ -144,7 +144,7 @@ public void TestLengthAndStarRatingUpdated()
             double lastStarRating = 0;
             double lastLength = 0;
 
-            AddStep("Add timing point", () => EditorBeatmap.ControlPointInfo.Add(500, new TimingControlPoint()));
+            AddStep("Add timing point", () => EditorBeatmap.ControlPointInfo.Add(200, new TimingControlPoint { BeatLength = 600 }));
```

Comment:
To save me going through the whole process, can you explain why this change was required?

Reply:
The test is placing 2 circles. Before this PR first one was placed on negative time, so they started to stack on the same beat. I changed this to make them placed on different beats to actually contribute to map's length.